app/channel_test.go

```diff
@@ -1898,3 +1898,23 @@ func TestClearChannelMembersCache(t *testing.T) {
 
 	th.App.ClearChannelMembersCache("channelID")
 }
+
+func TestGetMemberCountsByGroup(t *testing.T) {
+	th := SetupWithStoreMock(t)
+	defer th.TearDown()
+
+	mockStore := th.App.Srv().Store.(*mocks.Store)
+	mockChannelStore := mocks.ChannelStore{}
+	cmc := []*model.ChannelMemberCountByGroup{}
+	for i := 0; i < 200; i++ {
+		cmc = append(cmc, &model.ChannelMemberCountByGroup{
+			GroupId:                     "1",
```

Comment:
Typically, we generate ids using `model.NewId`

Reply:
Fixed.